src/playercards/CardsThatRedrawTokens.ttslua

```diff
@@ -0,0 +1,61 @@
+local turnOnHelper
+
+function onSave()
+  return JSON.encode(turnOnHelper)
+end
+
+function onLoad(savedData)
+  self.addContextMenuItem("Enable Helper", makeXMLButton)
+
+  if savedData and savedData ~= "" then
+    turnOnHelper = JSON.decode(savedData)
+    if turnOnHelper == true then
+      makeXMLButton()
+      self.clearContextMenu()
+      self.addContextMenuItem("Clear Helper", deleteButton)
+    end
+  end
+end
+
+function makeXMLButton()
+  turnOnHelper = true
+  self.clearContextMenu()
+  self.addContextMenuItem("Clear Helper", deleteButton)
+  log(color)
+  local xmlTable = {{
+      tag = "Button",
+      attributes = {
+          height = 450,
+          width = 1400,
+          rotation = rotation or "0 0 180",
+          scale = "0.1 0.1 1",
+          position = position or "0 -55 -22",
+          padding = "50 50 50 50",
+          font = "font_teutonic-arkham",
+          fontSize = fontSize or 250,
+          onClick = "triggerXMLTokenLabelCreation()",
+          color = color or "#77674DE6",
+          textColor = "White"
+      },
+      value = value or "Redraw Token"
+    }}
+  if icon then
+      xmlTable[1].attributes.iconWidth = "400"
+      xmlTable[1].attributes.iconAlignment = "Right"
+      xmlTable[1].attributes.icon = icon
+  end
+  self.UI.setXmlTable(xmlTable)
+end
+
+function triggerXMLTokenLabelCreation()
+    Global.call("activeRedrawEffect", originParams)
+    Global.call("makeButtonsToRedraw")
```

Comment:
I assume these functions are always gonna be used together? In that case they should get combined (or at least let the first one call the second one)